### ISSUE TITLE
Test Automation IDs for Login workflow

### ISSUE THREAD

**Issue by zdmc23** [DiscipleTools/disciple-tools-mobile-app#512]
Add Test Automation IDs in support of automating the testing of the Login workflow (PoC, and then rollout throughout app)

This also kicks off the new effort around Accessibility...

Since we're already relying on `accessibilityLabel` properties as Test Automation IDs (per FB recommendation: https://reactnative.dev/docs/testing-overview#testing-user-interactions), then we should get this project underway (prob should have been doing this from the jump)

https://reactnative.dev/docs/accessibility

**Comment by zdmc23** [comment]
See also: 
- https://appiumpro.com/editions/76-testing-react-native-apps-with-appium
- https://github.com/facebook/react-native/pull/9942
- https://93days.me/testing-react-native-application/
- https://testingbot.com/blog/react-native-testing
